driver/compmisc.ml

```diff
@@ -19,6 +19,29 @@
    then the standard library directory (unless the -nostdlib option is given).
  *)
 
+let init_path ?(dir="") () =
+  let dirs =
+    if !Clflags.use_threads then "+threads" :: !Clflags.include_dirs
+    else
+      !Clflags.include_dirs
+  in
+  let dirs =
+    !Compenv.last_include_dirs @ dirs @ Config.flexdll_dirs @
+    !Compenv.first_include_dirs
+  in
+  let exp_dirs =
+    List.map (Misc.expand_directory Config.standard_library) dirs 
+  in
+  let tempname = 
+    if !Clflags.remove_cwd then 
+      List.rev_append exp_dirs (Clflags.std_include_dir ()) @ [dir]
```

Comment:
What are the semantics of the new flag? I would have imagined that if one passes `-removecwd` the current directory is *not* added to the load path at all, but if I understand correctly here, this flag means that the current directory *is* added, but at the end. Is this semantic useful? Wouldn't it be simpler to just remove it altogether?

Reply:
Thank you for your comments. I did not remove the current working directory (cwd) from the path. I tried to delete it but got an error (compiler can't find a .cmo file). So I guess it's a better solution to de-prioritize the cwd. 
The name of the flag `-removecwd` is not correct and I currently have no idea how to rename the flag